example/src/pages/support/FAQ/index.md

```diff
@@ -1,3 +1,11 @@
+---
+title: FAQ
+description: This is the FAQ page for Adobe Analytics
+contributor_name: Kalpesh Mehta from Corra
+contributor_link: https://partners.magento.com/portal/details/partner/id/70/
+edition: ee_only
```

Comment:
Per conversation in grooming today, let's drop "_only" from all options.

Reply:
Hmm I'm not certain how to tie in the github info if it's defined in the frontmatter portion. Currently the info is loaded in via  plugin (https://github.com/adobe/gatsby-source-github-file-contributors) then passed to Provider (https://github.com/adobe/aio-theme/blob/main/packages/gatsby-theme-aio/src/components/Layout/index.js#L185) and then injected into the page (https://github.com/adobe/aio-theme/blob/3f0a6fa59ae62314d9c72ff04044d9ad9fda56b6/packages/gatsby-theme-aio/src/components/MDXFilter/index.js#L348).

If I recall our conversation correctly, I think you wanted a way to define the contributors more by hand than the current github contributor plugin logic? So I can see having both and then you can reference the one you want.